_navbar.scss

```diff
@@ -0,0 +1,35 @@
+$mainColor: rgb(26, 27, 29);
+$effectColor: rgb(202, 128, 17);
+$mainWhite: #FEFFFF;
+$primaryFont: Sentinel SSm A,Sentinel SSm B,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol;
+
+nav {
+    ul {
+        font-family: $primaryFont;
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: center;
+        font-size: 1rem bold;
```

Comment:
This doesn't look right. `bold` doesn't seem like a valid value for font-size.

Reply:
Ok, fixed.